uint32 node_id
lifelong_mapping/RawMessage data
